waffle_hub/hub/hub.py

```diff
@@ -1064,6 +1081,8 @@ def inner(callback: TrainCallback, result: TrainResult):
                 field_value = getattr(
                     self.DEFAULT_PARAMS[self.task][self.model_type][self.model_size], k
                 )
+                if self.check_hpo_artifact():
+                    field_value = getattr(self.get_train_config(), k)
                 setattr(cfg, k, field_value)
```

Comment:
I think it sholud be changed like:
`if self.check_hpo_artifact(): ... else: (default)`

Reply:
ok!